### ISSUE TITLE
Dons't work on Windows 10

### ISSUE THREAD

**Issue by 81x08** [Jire/Charlatano#397]
Dons't work on Windows 10, all function cheats.

**Comment by MoofMonkey** [comment]
Is there some errors in console when you're building cheat? Running?

**Comment by welshyben** [comment]
Yeah it does.... Just not for you. Have you tried uninstalling Java, reinstalling. Building the cheats again, and running while you're in-game with a weapon out in your hands?

**Comment by darknayt** [comment]
Don't work now for me too , give me some error

**Comment by legendary22** [comment]
1)Reinstall JDK
2)Go to C/users/legendary(my pc name is legendary your might me different)
3)delete .gradle folder
4)RUN build.bat again
it will fix

**Comment by welshyben** [comment]
the default build doesn't work for me but a build some dude threw up on here does

**Comment by 81x08** [comment]
Errors in the assembly is not present. The latest version of the JDK
